Sources/PackageDescription/Version.swift

```diff
@@ -49,10 +49,9 @@ extension Version: Equatable {
 
 // MARK: Hashable
 
+//FIXME: Remove in 4.2
```

Comment:
This can be done now because PackageDescription does not need to be 4.0 compatible.